w5e1.py

```diff
@@ -0,0 +1,82 @@
+"w5e1" 
+import random
+import os
+
+# pokemon s1e1 -> Ash late to professor Oak so can't choose pokemon...
+from datetime import datetime, timedelta
+
+MONDAYS = 1
+
+
+def pokemon():
+    pokemons = ['Bulbasaur', 'Squirtle', 'Charmander']
+    starter_pokemon = random.choice(pokemons)
+    print(starter_pokemon)
+
+
+'''
+ This Is The Way
+ Write a function that gets a path to the folder,
+ and returns the list of all files that begin with the
+ "startsWith" letter sequence in that folder.
+'''
+
+
+def startsWith_files(folder, startsWith):
```

Comment:
give a better name - its not the folder, its the folder_path

Reply:
def starts_with_files(folder_path_to_search_in, starts_with):

Reply:
def search_files_starts_with_in_folder(folder_path_to_search_in, starts_with):